lang_GENERIC_base/AST_generic.ml

```diff
@@ -574,6 +575,7 @@ and operator =
   | NotMatch (* !~ Ruby less: could be desugared to Not RegexpMatch *)
   | Range (* .. or ..., Ruby, one arg can be nil for endless range *)
   | NotNullPostfix (* ! in Typescript, postfix operator *)
+  | Length (* Lua *)
```

Comment:
again, this could be represented as a regular function call no? it is a special operator?